docs/_includes/Introduction.md

```diff
@@ -23,61 +28,69 @@ changing the program’s type declarations, specifications, and statements.
 The easiest way to try out [Dafny is in your web browser at
 rise4fun](http://rise4fun.com/Dafny)[@Rise4fun:dafny]. Once you get a bit
 more serious, you may prefer to [download](http://dafny.codeplex.com/) it
-to run it on your machine. Although Dafny can be run from the command
-line (on Windows or other platforms), the preferred way to run it is in
-Microsoft Visual Studio 2012 (or newer) or using emacs, where the Dafny
+to run it on your machine. Dafny can be run from the command
+line (on Windows or other platforms) or from an IDE
+such as emacs or
+Microsoft Visual Studio 2012 (or newer), where the Dafny
 verifier runs in the background while the programmer is editing the
-program.
+program. An editor such as VSCode can provide syntax highlighting without
+the built-in verification.
 
 The Dafny verifier is powered
 by [Boogie](http://research.microsoft.com/boogie)
 [@Boogie:Architecture;@Leino:Boogie2-RefMan;@LeinoRuemmer:Boogie2]
 and [Z3](https://github.com/z3prover)[@deMouraBjorner:Z3:overview].
 
-From verified programs, the Dafny compiler produces code (`.dll` or
-`.exe`) for the .NET platform via intermediate C\# files. However, the
+From verified programs, the Dafny compiler can produce code for a number
+of different backends: javascript, go, java, or
+the .NET platform via intermediate C\# files. However, the
 facilities for interfacing with other .NET code are minimal.
```

Comment:
Let's list .NET first and capitalize the names of the others. Also, say there's partial support for C++ backend.

Replace the "However" sentence with something like "Each backend provides a basic Foreign Function Interface (called `:extern`). However, there is no automatic FFI generator, so `:extern` stubs need to be written by hand."

Reply:
Done